MapboxNavigation.xcodeproj/project.pbxproj

```diff
@@ -2803,7 +2815,7 @@
 			buildSettings = {
 				CODE_SIGN_IDENTITY = "iPhone Developer";
 				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "";
-				DEBUG_INFORMATION_FORMAT = dwarf;
+				DEBUG_INFORMATION_FORMAT = "dwarf-with-dsym";
```

Comment:
Can you split this and the `STRIP_SWIFT_SYMBOLS` change out into a separate PR if it’s intentional?

Reply:
Unintentional. Will revert.